package.json

```diff
@@ -27,12 +27,12 @@
     "apollo-server-micro": "^2.24.1",
     "framer-motion": "4",
     "graphql": "^15.5.0",
-    "next": "latest",
+    "next": "^11.0.0",
```

Comment:
I'm wondering why we had to change these?

Reply:
This change was a result of running `yarn` so the change was done automatically. I can revert this file if you'd like.

Reply:
Okay, I see. Let's leave it for now.